core/baseline-core-v1.0-psd01.md

```diff
@@ -1089,8 +1095,19 @@ Abstraction Layers utilized in a BPI MUST be compatible with widely used externa
 #### **[R62]**	
 Abstraction Layers utilized in a BPI MUST support roles & access management.
 
+Roles management means the a BPI has the ability to define different roles and associated attributes for BPI Subjects, for example, the role of a "Workgroup Administrator" together with associated attributes such as "Add Workgroup Participant" or "Remove Workgroup Participant".
```

Comment:
"Remove Workgroup Participant" feels a little harsh for the administrator (as we like to call them, the "Workgroup Operator").

Reply:
It is a CRUD operation ... and remove is a legit action